scripts/gen-jsonschema.mts

```diff
@@ -101,11 +131,28 @@ if (check) {
       process.exit(1);
     }
   }
+  // docs/assets/config-*.schema.json
+  for (const [filePath, schema] of Object.entries(configSchemas)) {
+    try {
+      const file = orderedStringify(JSON.parse(fs.readFileSync(filePath, 'utf8')));
+      if (file !== orderedStringify(schema)) {
+        console.error(`Mismatch in ${filePath} (Do you need to run \`make update-jsonschema\`?)`);
+        process.exit(1);
+      }
```

Comment:
Have you thought of using an object comparison instead of JSON comparison, to avoid problems with key order? Either using `_.isEqual`, or a custom recursive comparison function, or something like [`json-diff`](https://www.npmjs.com/package/json-diff) for a prettier diff?

Reply:
Is there a specific improvement that using `json-diff` would make over `orderedStringify`? It should reformat it correctly when you run `make update-jsonschema`.

Reply:
The main thing is that it makes it easier to find what the difference is. This is similar to the diff we see in the database descriptions, where `make update-database-description` could easily just regenerate the description, but seeing what the difference is can make it clearer in CI what changed that caused the requirement for a new update.

This is not as important here since there's a clear one-to-one mapping from Zod to schema (whereas in CI this applies to a set of migrations), but it still helps to know what changed.